adapter/sql/util.go

```diff
@@ -51,6 +63,19 @@ func Escape(config Config, field string) string {
 	return escapedField.(string)
 }
 
+func isAliasField(field string) (*aliasField, bool) {
+	if !strings.Contains(strings.ToLower(field), " as ") {
+		return nil, false
+	}
+
+	f := strings.Fields(field)
```

Comment:
I think we can just use Index (https://golang.org/pkg/strings/#Index) function, and then split and merge the string manually using the returned index, that way the lookup inside string only happened once, not twice (Contains, Fields)